psychopy/sound/backend_pyo.py

```diff
@@ -15,7 +15,18 @@
                                 NOT_STARTED, FOREVER)
 import os
 import io
-from contextlib import redirect_stdout
+try:
+    from contextlib import redirect_stdout
```

Comment:
Could you change the `try...except` to an `if...else`, checking for the Python version via `psychopy.constants.PY3`? That way it's more obvious why there's two different ways to handle the issue

Reply:
Done.